app/vmagent/main.go

```diff
@@ -261,7 +261,7 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 		path = strings.TrimSuffix(path, "/")
 	}
 	switch path {
-	case "/prometheus/api/v1/write", "/api/v1/write":
+	case "/prometheus/api/v1/write", "/api/v1/write", "/api/v1/push":
```

Comment:
Seems like `/api/v1/push` could potentially conflict with Loki ingestion path, if we ever will add its support to vmagent.
Adding a prefix `prometheus/api/v1/push` would help though.

Workarounds are the following:
1. Configure pushing clients to use [standard Prometheus path](https://prometheus.io/docs/prometheus/latest/querying/api/#remote-write-receiver) `/api/v1/push`
2. Use vmauth to reroute requests to the correct path.

Reply:
Makes sense, thanks